atlasdb-feign/src/main/java/com/palantir/atlasdb/http/FeignOkHttpClients.java

```diff
@@ -39,6 +41,9 @@
 public final class FeignOkHttpClients {
     @VisibleForTesting
     static final String USER_AGENT_HEADER = "User-Agent";
```

Comment:
While we're editing this file, is there an HttpHeaders.USER_AGENT that we can use here?